packages/core/src/odata-v2/request/odata-function-import-request-config.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  ODataUri,
+  FunctionImportParameter,
+  FunctionImportParameters,
+  RequestMethodType,
+  ODataFunctionImportRequestConfig
+} from '../../odata-common';
+
+export class ODataFunctionImportRequestConfigV2<
+  ParametersT
+> extends ODataFunctionImportRequestConfig<ParametersT> {
+  /**
+   * Creates an instance of ODataFunctionImportRequestConfig.
+   *
+   * @param method - HTTP method for the request
+   * @param defaultServicePath - Default path of the service
+   * @param functionImportName - The name of the function import.
+   * @param parameters - Object containing the parameters with a value and additional meta information
+   * @param oDataUri - ???
+   */
+  constructor(
+    method: RequestMethodType,
+    defaultServicePath: string,
+    functionImportName: string,
+    parameters: FunctionImportParameters<ParametersT>,
+    oDataUri: ODataUri
+  ) {
+    super(method, defaultServicePath, functionImportName, parameters, oDataUri);
+  }
+
+  resourcePath(): string {
+    return this.functionImportName;
+  }
+
+  queryParameters(): Record<string, any> {
```

Comment:
I guess for v2, we keep the same as the previous common

Reply:
yes, the new v2 is basically a copy of the file that was in common.